python/src/space/core/storage.py

```diff
@@ -53,10 +53,12 @@ class Storage(paths.StoragePathsMixin):
   Not thread safe.
   """
 
-  def __init__(self, location: str, metadata: meta.StorageMetadata):
+  def __init__(self, location: str, cur_metadata_file: str,
```

Comment:
nit: can we rename "cur_metadata_file" to simply "metadata_file"?

Reply:
nit: maybe `metadata_path` and `self._metadata_path`, to match with the existing `new_metadata_path` var